src/project/storage/mod.rs

```diff
@@ -62,7 +62,13 @@ impl ProjectStorage {
     pub async fn set(&self, id: &str, data: &ProjectDataResult) {
         let cache_key = build_cache_key(id);
 
-        let serialized = crate::storage::serialize(&data).unwrap(); //?;
+        let serialized = match crate::storage::serialize(&data) {
+            Ok(serialized) => serialized,
+            Err(err) => {
+                warn!(?err, "failed to serialize cached project data");
```

Comment:
Being unable to serialize well-known data is very bad

```suggestion
                error!(?err, "failed to serialize cached project data");
```